launchdarkly_rum/manifest.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Looks like this is just a copy of the same manifest.json for [`launchdarkly`](https://github.com/DataDog/integrations-extras/blob/95b8ce0128531e87869edf5e9b866ed8e0f7e898/launchdarkly/manifest.json). Can you [generate a new tile integration](https://datadoghq.dev/integrations-core/ddev/cli/#ddev-create) so that the manifest.json can be generated?